### PR TITLE
add External DB with All Recipes and [id]-Info

### PR DESCRIPTION
**Pull request by Larissartoricader**

### CODE REVIEW

File: pages/api/recipes/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import dbConnect from "@/db/connect";
+import Recipe from "@/db/models/Recipe";
+
+export default async function handler(request, response) {
+  try {
+    await dbConnect();
+    if (request.method === "GET") {
+      const recipe = await Recipe.find();
```
Comment (Roland-Hufnagel) [inline pages/api/recipes/index.js:8]: in this route I would name the variable `recipes` because I expect them to be multiple.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi. You implemented two endpoints but I cannot see any usage of these endpoints. So it doesn't fit the value proposition of the userstory.

File: pages/api/recipes/index.js (hunk 1 above)
Comment (Larissartoricader) [reply pages/api/recipes/index.js:8]: Done! Thank you.

File: db/models/Recipe.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  ingredients: [{ type: String, required: true }],
+  preparation: { type: String, required: true },
+  usage: { type: String, required: true },
+  symptoms: [{ type: String, required: true }],
+});
+
+const Recipe =
+  mongoose.models.initial_recipes ||
+  mongoose.model("initial_recipes", RecipeSchema, "initial_recipes");
+console.log(Recipe);
```
[the remark this answers is not in the document]
Comment (Larissartoricader) [reply db/models/Recipe.js:16]: Done.

**Comment by Larissartoricader** [comment]
@Roland-Hufnagel 

> Hi. You implemented two endpoints but I cannot see any usage of these endpoints. So it doesn't fit the value proposition of the userstory.

@Roland-Hufnagel Thank you for your feedback. You're 100% right, but I split the backend-implementation in 2 UserStories: In this  first UserStory I want only the create the api/recipes /api/recipes/[id] and in the [second one](https://github.com/users/Larissartoricader/projects/2/views/1?pane=issue&itemId=61798156) I will use the api and connect the end points with the features that we already have.

**Comment by MareikeFla** [comment]
I can not reach the API yet, but as far as I understood this is because there are two US. 
LGTM

**Pull request merged**